bindings/matrix-sdk-ffi/src/encryption.rs

```diff
@@ -413,16 +414,51 @@ impl Encryption {
 
     /// Get the E2EE identity of a user.
     ///
-    /// Returns Ok(None) if this user does not exist.
+    /// This method always tries to fetch the identity from the store, which we
+    /// only have if the user is tracked, meaning that we are both members
+    /// of the same encrypted room. Optionally it can request the user from the
+    /// homeserver if not found locally.
     ///
-    /// Returns an error if there was a problem contacting the crypto store, or
-    /// if our client is not logged in.
-    pub async fn get_user_identity(
+    /// # Arguments
+    ///
+    /// * `user_id` - The ID of the user that the identity belongs to.
+    ///
+    /// * `request_from_homeserver_if_needed` - If true, the identity will be
+    ///   requested from the homeserver if it is not found in the crypto store.
+    ///   When `true` the identity returned is guaranteed to be up-to-date.
```

Comment:
> When `true` the identity returned is guaranteed to be up-to-date.

I would claim that if when `true` there is **always** a request sent to the server, no? I mean in case identity is know locally, and has changed without the SDK knowing (no network, etc.), this assumption will not be true.

Reply:
How would the app / the SDK know whether an identity was known locally but we should trigger the request because it was updated? In that case, we might as well spawn the request all the time, since we can't really know when it's been updated, right?

Reply:
https://github.com/matrix-org/matrix-rust-sdk/commit/195fed7f5c1f60dc8ccc50ef92e7ccca6189f446